ash/build.rs

```diff
@@ -0,0 +1,24 @@
+fn main() {
+    #[cfg(feature = "linked")]
+    {
+        use std::env;
+
+        let target_family = env::var("CARGO_CFG_TARGET_FAMILY").unwrap();
+        let target_pointer_width = env::var("CARGO_CFG_TARGET_POINTER_WIDTH").unwrap();
+
+        println!("cargo:rerun-if-env-changed=VULKAN_SDK");
+        if let Ok(var) = env::var("VULKAN_SDK") {
+            let suffix = match (&*target_family, &*target_pointer_width) {
+                ("windows", "32") => "Lib32",
+                ("windows", "64") => "Lib",
+                _ => "lib",
+            };
+            println!("cargo:rustc-link-search={}/{}", var, suffix);
+        }
+        let lib = match &*target_family {
+            "windows" => "vulkan-1",
+            _ => "vulkan",
+        };
+        println!("cargo:rustc-link-lib={}", lib);
```

Comment:
Can you remind me what kind of linking this is? Will this always be static linking?

Reply:
fwiw only having static linking is fine I think, I don't think we need dynamic linking because we have runtime loading which are fairly similar.

Reply:
Per [rustc docs](https://doc.rust-lang.org/rustc/command-line-arguments.html#option-l-link-lib), this will link whatever type of library is available, favoring dynamic if both are found. Note that the Vulkan loader is normally distributed only as a dynamic library.

> I think we can keep ash-molten as is and it can internally do `Entry::from_static_fn`. I think we can use no-default-features there as well and let ash-molten statically link with vulkan.

This should work great.